extended/src/main/java/io/kubernetes/client/extended/leaderelection/LeaderElector.java

```diff
@@ -368,7 +366,13 @@ private void maybeReportTransition() {
     this.reportedLeader = this.observedRecord.getHolderIdentity();
 
     if (this.onNewLeaderHook != null) {
-      this.hookWorkers.submit(() -> onNewLeaderHook.accept(this.reportedLeader));
+      this.hookWorkers.submit(
+          () -> {
+            log.info(
```

Comment:
@brendandburns i moved info logging to where the leader-transition hook is called